### DISCUSSION TITLE
Clear out finished mangas

### DISCUSSION THREAD

**Discussion by librafrog** [manga-download/haruneko#54] [Ideas]
- [ ] The ability to delete finished graphic novels after we are done with them and let the program know they are not present. By time or manually

**Comment by Sheepux** [comment]
You mean just un-bookmark or delete from disk ? 
Could be a combination of both : when removing from bookmark it will ask if you want to remove the downloaded content.
But I thing that's more a new feature request that should be created in hakuneko's github issues (not really haru specific).

**Reply by librafrog** [reply]
Yep, that is the idea - to be able to delete it from disk. The exact functionality can be decided later.
